src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,88 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final String DATE = "2019-09-10";
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+        MovieService movieService =
+                (MovieService) injector.getInstance(MovieService.class);
+        CinemaHallService cinemaHallService =
+                (CinemaHallService) injector.getInstance(CinemaHallService.class);
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
 
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
+
+        Movie united93 = new Movie("United 93");
+        united93.setDescription("A real-time account of the events on United Flight 93.");
+        movieService.add(united93);
+
+        Movie neerja = new Movie("Neerja");
+        neerja.setDescription("A biographical thriller film, based on a real-life event.");
+        movieService.add(neerja);
+
+        Movie endOfWatch = new Movie("End of Watch");
+        endOfWatch.setDescription("An action film about Los Angeles Police Department officers.");
+        movieService.add(endOfWatch);
+
+        Movie spotlight = new Movie("Spotlight");
+        spotlight.setDescription("A biographical drama film about investigative journalist unit.");
```

Comment:
The `spotlight` movie is created but not added to the `movieService`. Consider adding it using `movieService.add(spotlight);` to ensure it is stored and can be retrieved later.